pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -177,6 +177,10 @@ func (plugin *kubenetNetworkPlugin) Name() string {
 	return KubenetPluginName
 }
 
+func (plugin *kubenetNetworkPlugin) Capabilities() uint {
```

Comment:
@bprashanth I think this approach is more "magic", unless I don't quite understand what you're suggesting.  If the goal is to get rid of Capabilities(), I think it's better to have a well-defined API between the plugins and the core and keep an explicit method.  Getting rid of Capabilities() and embedding the capabilities in the plugin type would mean every plugin has to embed the capabilities and we lose some of the benefit of plugins being an interface.  Does that make sense?